### PR TITLE
Implement "generate password" option for signup forms

### PR DESCRIPTION
**Pull request by PalmerAL**
https://github.com/minbrowser/min/assets/10314059/c1bfe2ee-4ac1-4ed5-83ad-9596491c4b9f

### CODE REVIEW

File: localization/languages/ru.json [hunk 1]
```diff
@@ -290,6 +290,7 @@
     "passwordCaptureSave": "Сохранить",
     "passwordCaptureDontSave": "Не сохранять",
     "passwordCaptureNeverSave": "Никогда не сохранять",
+    "generatePassword": null, // missing translation
```
Comment (Sestowner) [inline localization/languages/ru.json:293]:
```suggestion
    "generatePassword": "Сгенерировать пароль",
```

File: localization/languages/uk.json [hunk 2]
```diff
@@ -291,6 +291,7 @@
     "passwordCaptureSave": "Зберегти",
     "passwordCaptureDontSave": "Не зберігати",
     "passwordCaptureNeverSave": "Ніколи не зберігати",
+    "generatePassword": null, // missing translation
```
Comment (Sestowner) [inline localization/languages/uk.json:294]:
```suggestion
    "generatePassword": "Згенерувати пароль",
```

**Comment by gribanoveu** [comment]
Is it possible to add it to be entered in two fields at once? Password and confirm password

**Comment by PalmerAL** [comment]
@gribanoveu It's possible; we need to write a method for detecting confirmation fields. I'll look into it at some point, or you're welcome to contribute one if you want.

**Comment by PalmerAL** [comment]
@gribanoveu I've implemented this in the latest commit; if you happen to have a chance to test it out, let me know if you see any issues!

**Pull request merged**